src/subdomains/generic/user/services/webhook/dto/account-merge-webhook.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { WebhookDto, WebhookType } from './webhook.dto';
+
+export class AccountMergeWebhookData {
+  @ApiProperty()
+  master: number;
+
+  @ApiProperty()
+  slave: number;
+}
+
```

Comment:
Yeah I thought master and slave are familiar words in development but we could also call them old and new

Reply:
Slave doesn't make sense IMO since the slave doesn't have any functionality after the merge ... it's just gone 💥 maybe we should even not include the slave ID in the data and set something like this:
```ts
export class AccountMergeWebhookData {
  @ApiProperty()
  accountId: number;
}
```
Then we could name it `AccountChanged` and just add some further data to this DTO, if there is the need for it in the future.
WDYT?

Reply:
But then I don't know what the id was before the merge? 
That would be confusing too or?